Make Cypher 25 the default query language (#1483)
neo4j/docs-cypher: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

Replacement syntax for deprecated and removed features are also indicated.

[IMPORTANT]

> Cypher 25 is the default Cypher version in Neo4j 2026.02+ databases.

Author: ```suggestion
Neo4j 2026.02+ databases explicitly set Cypher 25 as their query language.
```

Cypher 25 was introduced in Neo4j 2025.06 and can only be used on Neo4j 2025.06+ databases.
Features removed in Cypher 25 are still available on Neo4j 2025.06+ databases either by prepending a query with `CYPHER 5` or by having Cypher 5 as the default language for the database.
For more information, see xref:queries/select-version.adoc[].

[[cypher-deprecations-additions-removals-2026.02]]
== Neo4j 2026.02

=== New in Cypher 25

[cols="2", options="header"]
|===
| Feature
| Details

a|
label:functionality[]
label:new[]

> Cyper 25 is now the default Cypher version for new databases.

Author: ```suggestion
New databases now explicitly set Cyper 25 as their query language.
```

> a| Cyper 25 is the default Cypher version for new self-managed installations and new Aura instances.

Author: ```suggestion
a| New self-managed installations and new Aura instances now explicitly set Cyper 25 as their query language.
```

New self-managed deployments have the setting `db.query.default_language=CYPHER_25` in their _neo4j.conf_ file.
Existing deployments retain their _neo4j.conf_ file.

> The setting can be changed manually for both, see xref:queries/select-version.adoc#select-default-cypher-version[Select the default Cypher version for a database].

Author: ```suggestion
Both can be adjusted via the setting link:{neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_db.query.default_language[`db.query.default_language`].
```
